### PR TITLE
Release 0.8.0

### LINKED ISSUE #1814: I think we're in Beta not Alpha
## URL or Command Line Option with the issue being reported

https://docs.sqlfluff.com/en/stable/index.html

> Note
> SQLFluff is still in an open alpha phase - expect the tool to change significantly over the coming months, and expect potentially non-backward compatible api changes to happen at any point. In particular:

## Issue/Suggested Improvement

Other places suggest we're in Beta not Alpha.

@WittierDinosaur could you update as part of 0.8.0 PR?

### PR DESCRIPTION
**Pull request by WittierDinosaur**
### Brief summary of the change made
Release 0.8.0

Fixes #1814

### Are there any other side effects of this change that we should be aware of?
No

### CODE REVIEW

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:16]:
```suggestion
This release brings an improvement to the performance of the parser, and a rebuild of the Jinja Templater. Lots of dialect improvements have also been done. Full list of changes below:
```

I'd avoid use of the word minor since < 1.0.0 the mid-point "minor" versions are used for majors too.

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:23]:
```suggestion
* Hive: INSERT statement support [#1828](https://github.com/sqlfluff/sqlfluff/pull/1828) [@mifercre](https://github.com/mifercre)
```

Will move the temp external table to bugs fix section.

(I've edited the issues too so Release notes will be updated)

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:28]:
```suggestion
* L029: Optionally check quoted identifiers in addition to naked identifiers [#1775](https://github.com/sqlfluff/sqlfluff/pull/1775) [@jpers36](https://github.com/jpers36)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:35]:
```suggestion
* TSQL - allow optional brackets for column default constraints [#1760](https://github.com/sqlfluff/sqlfluff/pull/1760) [@nevado](https://github.com/nevado)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:37]:
```suggestion
* Postgres: Added support for double precision [#1764](https://github.com/sqlfluff/sqlfluff/pull/1764) [@WittierDinosaur](https://github.com/WittierDinosaur)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:40]:
```suggestion
* TSQL - add IDENTITY column constraint [#1757](https://github.com/sqlfluff/sqlfluff/pull/1757) [@nevado](https://github.com/nevado)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:48]:
```suggestion
* Fix: Add missing init file to sqlfluff.core.templaters.slicers [#1826](https://github.com/sqlfluff/sqlfluff/pull/1826) [@CrossNox](https://github.com/CrossNox)
* Hive: Fix order of CREATE TEMPORARY EXTERNAL TABLE [#1825](https://github.com/sqlfluff/sqlfluff/pull/1825) [@mifercre](https://github.com/mifercre)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:49]:
```suggestion
* TSQL add AS keyword as optional in PIVOT-UNPIVOT [#1807](https://github.com/sqlfluff/sqlfluff/pull/1807) [@tkachenkomaria244](https://github.com/tkachenkomaria244)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:50]:
```suggestion
* Prevent L019 plus L034 corrupting SQL [#1803](https://github.com/sqlfluff/sqlfluff/pull/1803) [@barrywhart](https://github.com/barrywhart)
```

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:12]: Maybe go tomorrow to give people the chance to merge those last few PRs? Let's play it by ear and see.

```suggestion
## [0.8.0] - 2021-11-07
```

File: src/sqlfluff/config.ini
Comment (tunetheweb) [inline src/sqlfluff/config.ini:4]: Annoyingly this also needs to be done in `plugins/sqlfluff-templator-dbt`. Both get released at once and pypi will complain about releasing an already released version again. @alanmcruickshank did try to make them dependent on the same config file but couldn't get it working :-( Eventually this might be pulled out to another repo so they can be released independently.

**Review by tunetheweb — CHANGES_REQUESTED** [review]
A few changes.

Also could we include a fix for #1814 in this?

And also could we include a line in `docs/index.rst` as we do for minor bumps?

**Comment by WittierDinosaur** [comment]
Think I've hit everything

File: CHANGELOG.md
Comment (tunetheweb) [inline CHANGELOG.md:20]: Why did we remove this one?

Comment (WittierDinosaur) [reply CHANGELOG.md:20]: ...Oops

File: docs/source/index.rst
Comment (tunetheweb) [inline docs/source/index.rst:14]: Let's soften this a bit:

```suggestion
    **SQLFluff** is still in an open Beta phase - expect potentially
    non-backward compatible api changes to happen with releases
    (particularly when major or minor version numbers increase).
    Notable changes:
```

**Comment by tunetheweb** [comment]
Could you also add #1776 that I merged?

**Comment by WittierDinosaur** [comment]
Done

**Review by tunetheweb — APPROVED** [review]
Approving, so you can merge at your pleasure but suggest we wait to double check if @barrywhart and @adam-tokarski 's changes can get in here too.

@barrywhart did originally mention he wanted #1788 to wait until 0.8.1 but let's check that's still the case.
I've approved #1609 from @adam-tokarski but just want to double check with him it's good to go

If you have any time until then @WittierDinosaur, I would love your thoughts on both?

@iajoiner I see you also have #1806 open so let us know ASAP if you plan to have that complete today to include in this release.

**Comment by WittierDinosaur** [comment]
I'll wait until tomorrow to release - just incase anyone wants to chuck anything else in. 99% sure @barrywhart doesn't want the dbt stuff in this release. With the progress bar, happy for that to go in.

**Comment by barrywhart** [comment]
I'm okay either way with respect to my PR. Probably smarter to wait, but it's an exciting change, so 🤷. Alan still hasn't approved the PR, anyway. Definitely don't wait on my PR. The release must roll! 🎉

**Comment by adam-tokarski** [comment]
I don't see any further issues with progressbar, so it can be merged if you ask me.

**Comment by tunetheweb** [comment]
@WittierDinosaur can you add these:
- Updated L009 logic to only allow a single trailing newline. (#1838) @jpy-git
- Progressbar utility (#1609) @adam-tokarski
- Teradata dialect: Add support for SEL form of SELECT (#1776) @samlader

Let's also add the Progressbar one to the release summary and index.rst as quite a big change.

Then I think we're good to go!

**Comment by WittierDinosaur** [comment]
@tunetheweb added. Can you give it the final nod before I merge?

**Review by tunetheweb — APPROVED** [review]
Go for it @WittierDinosaur ! 🚀

Thanks for taking care of the admin this time round. Good to spread the knowledge.

**Pull request merged**
